core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -39,7 +40,17 @@ object BrokerApiVersionsCommand {
   def execute(args: Array[String], out: PrintStream): Unit = {
     val opts = new BrokerVersionCommandOptions(args)
     val adminClient = createAdminClient(opts)
-    val brokerMap = adminClient.listAllBrokerVersionInfo()
+
+    // Wait until AdminClient#findAllBrokers returns a non-empty list of brokers
```

Comment:
Maybe we could move this to a separate method in `AdminClient` or a helper in `TestUtils`. Something like `awaitBrokers`?